File: sdk/dart/lib/model/payment.dart
part of swagger.api;


@Entity()
class Payment {
  
  @Property(name: 'ID')
  String ID = null;
  

  @Property(name: 'Type')
  String type = null;
  

  @Property(name: 'CreditCardID')
  String creditCardID = null;
  

  @Property(name: 'SpendingAccountID')
  String spendingAccountID = null;
  

  @Property(name: 'Description')
  String description = null;
  

  @Property(name: 'Amount')
  Number amount = null;
  

  @Property(name: 'xp')
  Object xp = null;
  
  Payment();

  @override
  String toString()  {
    return 'Payment[ID=$ID, type=$type, creditCardID=$creditCardID, spendingAccountID=$spendingAccountID, description=$description, amount=$amount, xp=$xp, ]';
  }

}

File: sdk/javascript-closure-angular/API/Client/Payment.js
goog.provide('API.Client.Payment');

/**
 * @record
 */
API.Client.Payment = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.Payment.prototype.ID;

/**
 * @type {!string}
 * @export
 */
API.Client.Payment.prototype.type;

/**
 * @type {!string}
 * @export
 */
API.Client.Payment.prototype.creditCardID;

/**
 * @type {!string}
 * @export
 */
API.Client.Payment.prototype.spendingAccountID;

/**
 * @type {!string}
 * @export
 */
API.Client.Payment.prototype.description;

/**
 * @type {!number}
 * @export
 */
API.Client.Payment.prototype.amount;

/**
 * @type {!API.Client.Object}
 * @export
 */
API.Client.Payment.prototype.xp;

